corporate/models.py

```diff
@@ -113,6 +127,18 @@ def get_current_plan_by_realm(realm: Realm) -> Optional[CustomerPlan]:
 
 
 class LicenseLedger(models.Model):
+    """
+    This table's purpose is to store the current, and historical,
+    count of "seats" purchased by the organization.
+
+    Because we want to keep historical data, when the purchased
+    seat count changes, a new LicenseLedger object is created,
+    instead of updating the old one. This lets us preserve
+    the entire history of how the seat count changes, which is
+    important for analytics as well as auditing and debugging
```

Comment:
Yes.  Also for charging the customers for the extra seats they purchased at the end of their billing month.

Reply:
@hackerkid Could you point me to the code where that calculation is done?

Reply:
Yup. That should be the `invoice_plan` function in stripe.py.